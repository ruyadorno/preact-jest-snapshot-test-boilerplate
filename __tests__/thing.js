const render = require('preact-render-to-string');
const Thing = require('../thing');

describe('Thing component', () => {
	it('should render with a provided name', () => {
		const tree = render(
			<Thing name="Lorem Ipsum" />
		);
		expect(tree).toMatchSnapshot();
	});
	it('should render a thing containing a link', () => {
		const tree = render(
			<Thing name="Lorem Ipsum" link="https://mobile.twitter.com"/>
		);
		expect(tree).toMatchSnapshot();
	});
});

