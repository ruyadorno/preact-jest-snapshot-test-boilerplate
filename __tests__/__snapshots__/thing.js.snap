// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thing component should render a thing containing a link 1`] = `<a href="https://mobile.twitter.com">Lorem Ipsum</a>`;

exports[`Thing component should render with a provided name 1`] = `<a>Lorem Ipsum</a>`;
